src/SetUnits.jl

```diff
@@ -1,47 +1,57 @@
-
 # include("PhysicalConstants.jl")
```

Comment:
includes should be suppressed in the final version, as everything is included from AtomicAndPhysicalConstants.jl
if they're left in subsidiary files, I think they throw errors.